tasks/lib.yml

```diff
@@ -12,4 +12,18 @@
     - libselinux-python
     - ncurses-devel
     - openssl-devel
+  when: ansible_os_family == 'RedHat'
 
+- name: install dependent packages for debian derivative distros
+  apt: name={{item}} state=present update_cache=yes
```

Comment:
Updating the cache when installing packages helps with an issue of apt cache being stale for a newly created EC2 instance.